starboard/android/shared/drm_system.h

```diff
@@ -29,14 +29,14 @@
 #include "starboard/common/atomic.h"
 #include "starboard/common/log.h"
 #include "starboard/common/mutex.h"
-#include "starboard/common/thread.h"
+#include "starboard/shared/starboard/player/job_thread.h"
 #include "starboard/types.h"
 
 namespace starboard {
 namespace android {
 namespace shared {
 
-class DrmSystem : public ::SbDrmSystemPrivate, private Thread {
+class DrmSystem : public ::SbDrmSystemPrivate {
```

Comment:
Shall we add a switch (and keep this default to disabled), so we can experiment in production?